pkg/apis/v1alpha1/networkinfo_types.go

```diff
@@ -0,0 +1,56 @@
+/* Copyright © 2024 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+//+kubebuilder:object:root=true
+//+kubebuilder:storageversion
+
+// NetworkInfo defines network infos for a namespace.
+type NetworkInfo struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   NetworkInfoSpec   `json:"spec,omitempty"`
+	Status NetworkInfoStatus `json:"status,omitempty"`
```

Comment:
I feel we could put all fields directly in the NetworkInfo struct as top level fields. Could you check how we define NetworkInfo or LB Monitor CRDs before?

Reply:
I checked in LB Monitor CRD, it is defined as top level fields. So changed to top level.